sklearn/metrics/tests/test_score_objects.py

```diff
@@ -543,22 +541,6 @@ def test_thresholded_scorers_multilabel_indicator_data():
     score2 = roc_auc_score(y_test, np.vstack([p[:, -1] for p in y_proba]).T)
     assert_almost_equal(score1, score2)
 
-    # Multi-output multi-class decision_function
```

Comment:
We decided to remove this use-case in the last PR for `_get_response_values` if I am not wrong.